app/javascript/admin/controllers/snackbar_controller.js

```diff
@@ -0,0 +1,31 @@
+import { Controller } from 'stimulus';
+
+// Wraps the Preact Snackbar component into a Stimulus controller
+export default class SnackbarController extends Controller {
+  static targets = ['snackZone'];
+
+  async connect() {
+    const [{ h, render }, { Snackbar }] = await Promise.all([
+      import('preact'),
+      // eslint-disable-next-line import/no-unresolved
+      import('Snackbar'),
+    ]);
+
+    render(<Snackbar lifespan="3" />, this.snackZoneTarget);
+  }
+
+  async disconnect() {
+    const { render } = await import('preact');
```

Comment:
Great suggestion, thank you!

Reply:
On another note, I'm not sure this can be done that way (controllers are mounted and unmounted, which means there's no survinving cache), unless we replicate the same mechanism inside the admin pack. I'll make a note for a future PR if that's okay